src/test/java/com/example/project/page_object/CheckoutPage.java

```diff
@@ -0,0 +1,53 @@
+package com.example.project.page_object;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class CheckoutPage {
+    private WebDriver driver;
+
+    @FindBy(xpath = "//input[@name=\"firstname\"]")
+    private WebElement firstName;
```

Comment:
please specify the element type in the name e.g. firstNameInput, searchButton, stateDropdown, etc